runtime/onert/backend/gpu_cl/open_cl/kernels/Util.cc

```diff
@@ -77,132 +77,133 @@ std::string GetCommonDefines(CalculationsPrecision precision)
   return result;
 }
 
-float4 GetMaskForLastPlane(int channels)
-{
-  float4 mask = float4(0.0f);
-  const int reminder = channels % 4 == 0 ? 4 : channels % 4;
-  for (int i = 0; i < reminder; ++i)
-  {
-    mask[i] = 1.0f;
-  }
-  return mask;
-}
+// float4 GetMaskForLastPlane(int channels)
+// {
+//   float4 mask = float4(0.0f);
+//   const int reminder = channels % 4 == 0 ? 4 : channels % 4;
+//   for (int i = 0; i < reminder; ++i)
+//   {
+//     mask[i] = 1.0f;
+//   }
+//   return mask;
+// }
 
-int3 GetFirstSuitableWorkGroup(const std::vector<int3> &wgs, int max_wg_size)
-{
-  for (const auto &wg : wgs)
-  {
-    const int wg_size = wg.x * wg.y * wg.z;
-    if (wg_size <= max_wg_size)
-    {
-      return wg;
-    }
-  }
-  return {1, 1, 1};
-}
+// int3 GetFirstSuitableWorkGroup(const std::vector<int3> &wgs, int max_wg_size)
+// {
+//   for (const auto &wg : wgs)
+//   {
+//     const int wg_size = wg.x * wg.y * wg.z;
+//     if (wg_size <= max_wg_size)
+//     {
+//       return wg;
+//     }
+//   }
+//   return {1, 1, 1};
+// }
```

Comment:
Any reason for commenting this w/o removing it ?

Reply:
Deleted codes are unnecessary codes or will be used as a whole.
But this code will be partially used. So it is commented

Reply:
How about useing `#if 0 ... #endif` ?